script/Deploy.s.sol

```diff
@@ -37,6 +35,45 @@ contract Deploy is BaseScript {
     }
 }
 
+contract DeployWakuRlnV2 is BaseScript {
+    function run() public broadcast returns (address) {
+        return address(deploy());
+    }
+
+    function deploy() public returns (WakuRlnV2) {
+        return new WakuRlnV2();
+    }
+}
+
+contract DeployProxy is BaseScript {
+    uint32 public constant MAX_TOTAL_RATELIMIT_PER_EPOCH = 160_000;
+    uint32 public constant MIN_RATELIMIT_PER_MEMBERSHIP = 20;
+    uint32 public constant MAX_RATELIMIT_PER_MEMBERSHIP = 600;
+    uint32 public constant EXPIRATION_TERM = 180 days;
+    uint32 public constant GRACE_PERIOD = 30 days;
+
+    function run() public broadcast returns (address) {
+        address priceCalcAddr = DevOpsTools.get_most_recent_deployment("LinearPriceCalculator", block.chainid);
+        address wakuRlnV2ImplAddr = DevOpsTools.get_most_recent_deployment("WakuRlnV2", block.chainid);
```

Comment:
This is cool, but what happens when you don't have a `broadcast` directory with a previous deployment on your machine?

Reply:
Hm, good point. 
I'll change the script to allow the usage of environment variables and/or indicate you want to use the broadcast directory.

Reply:
Change implemented in https://github.com/waku-org/waku-rlnv2-contract/pull/18/commits/6c9ce4da51d966f1926c95c2ae1f8a0ddc0a0944 
Now it will by default use the broadcast dir, unless env variables, for contracts that were already deployed.